upgrade_validation/hooks/install.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Select event from the default T-REX Profile
+  community.kubernetes.k8s_info:
+    api_version: v1
+    kind: Event
+    namespace: "{{ dci_openshift_app_ns|default('example-cnf') }}"
+    field_selectors:
+      - involvedObject.name = trex-app1
```

Comment:
Not sure if it's needed to be without spaces, i.e. `involvedObject.name=trex-app1`, let's test and confirm

Reply:
You were right! In fact I already faced a similar case here: https://github.com/dci-labs/trident-config/blob/master/hooks/deploy.yml#L130 and it is working with the extra spaces... weird